src/actions/profile-view.js

```diff
@@ -123,6 +132,34 @@ export function changeCallTreeSearchString(searchString: string): Action {
   };
 }
 
+function _addAllDescendants(
```

Comment:
Reading the code of the TreeView, it seems to me that it should still be its responsibility to know how to traverse a tree and determine what descendants it should expand. So rather than extracting that knowledge out of the component, I didn't change that code but copied it here and modified it a bit for the expand all action. So unfortunately there's a bit duplicated logic now.

Reply:
I think it would make sense to have a `getAllDescendants(childId) => Set` method in a tree. Then it's possible to add the expandedNodesIndexes to this Set after the call. And we could use it both from here and from TreeView.

What do you think ?

Reply:
On a side note, I think the "CallTree" component has too much logic, toggle could be an action. But not for this patch :)